spec/beaker/options/parser_spec.rb

```diff
@@ -283,6 +283,31 @@ def mock_out_parsing
 
             expect( options_test[:hosts_file_generated] ).to be true
           end
+
+          it 'beaker-hostgenerator failures trigger nice prints & a rethrow' do
+            options_test = {}
+            parser.instance_variable_set( :@options, options_test )
+            allow( Beaker::Options::HostsFileParser ).to receive(
+              :parse_hosts_file
+            ).and_raise( Errno::ENOENT )
+
+            mock_beaker_hostgenerator_cli = Object.new
+            allow( mock_beaker_hostgenerator_cli ).to receive( :execute )
+            allow( BeakerHostGenerator::CLI ).to receive(
+              :new
+            ).and_raise( BeakerHostGenerator::Exceptions::InvalidNodeSpecError )
```

Comment:
Does `BeakerHostGenerator::CLI.new` raise errors?  I don't think it should; `execute` is when the parsing occurs.